.vortex/tests/phpunit/Functional/AhoyWorkflowTest.php

```
@@ -222,6 +222,15 @@ public function testAhoyWorkflowProfileStandard(): void {
     $this->prepareSut();
     $this->adjustAhoyForUnmountedVolumes();
 
+    // TEMP DEBUG (revert): dump the theme build dir and injected env
+    // immediately before and after profile provisioning.
+    File::replaceContentInFile('.ahoy.yml',
+      'VORTEX_SHOW_LOGIN=0 ahoy provision  # Provision the site.',
+      'ahoy cli "echo DBG-BEFORE-PROV:; ls -la web/themes/custom/star_wars/build/css 2>&1 || echo DBG-MISSING-BEFORE; echo DBG-ENV:; env | grep -E \'FRONTEND|DRUPAL_THEME|WEBROOT|COMPOSE_|VOLUME|VORTEX_DEV|VORTEX_PROVISION|NODE\' | sort"' . "\n" .
+      '      VORTEX_SHOW_LOGIN=0 ahoy provision  # Provision the site.' . "\n" .
+      '      ahoy cli "echo DBG-AFTER-PROV:; ls -la web/themes/custom/star_wars/build/css 2>&1 || echo DBG-MISSING-AFTER"'
+    );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\AlexSkrypnyk\File\File' in method 'testAhoyWorkflowProfileStandard'. (undefined)

(StaticAccess)